apps/oozie/src/oozie/views/dashboard.py

```diff
@@ -1187,7 +1187,7 @@ def check_job_edition_permission(oozie_job, user):
 
 
 def has_job_edition_permission(oozie_job, user):
-  return user.is_superuser or oozie_job.user == user.username or (oozie_job.group and user.groups.filter(name=oozie_job.group).exists())
+  return user.is_superuser or oozie_job.user == user.username or (oozie_job.group and user.groups.filter(name=oozie_job.group).exists()) or (user.username in oozie_job.acl.split(','))
```

Comment:
(oozie_job.acl and user.username in oozie_job.acl.split(','))

?

when I tested I can get None for oozie_job.acl

Reply:
You are right. Fixed it.